src/chocolatey.tests.integration/scenarios/ListScenarios.cs

```diff
@@ -1,5 +1,4 @@
-// Copyright © 2017 - 2021 Chocolatey Software, Inc
-// Copyright © 2011 - 2017 RealDimensions Software, LLC
+﻿// Copyright © 2023-Present Chocolatey Software, Inc

```

Comment:
Is there a reason this copyright is changing the way it is?

Reply:
Can we even write "Present" and have it considered in any way legally valid? I usually see explicit years / dates in copyright notices.